content/user-research/analysing-user-research/decide.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: text/textblock
 ---
-You can use your findings to make decisions about what to work on, change or research next. This supports the agile method of planning continuously based on new facts or requirements.
+You can use your findings to make decisions about what to work on, change or research next. This supports the agile method of continuous planning with new facts or requirements.
```

Comment:
This reads more clearly. Not currently on prod. Recommend accepting this change.